package.json

```diff
@@ -76,7 +76,7 @@
         "test:vagrantBrowser": "vagrant ssh -c 'cd /home/vagrant/sync/universal; DISPLAY=:0; npm run test:browser'",
         "test:vagrantNode": "vagrant ssh -c 'cd /home/vagrant/sync/universal; npm run test:node'",
         "test:vagrantProduction": "vagrant ssh -c 'cd /home/vagrant/sync/universal; DISPLAY=:0; ./scripts/vagrantCloudBasedContainers.sh'",
-        "test:productionConfig": "node tests/production/AddUserSettingsToCouchTests.js && node tests/production/all-tests.js && node tests/production/DeleteUserSettingsFromCouchTests.js",
+        "test:productionConfig": "node tests/production/AddUserSettingsToCouchTests.js && node tests/production/all-tests.js && node tests/production/DeleteUserSettingsFromCouchTests.js && node tests/production/SolutionsRegistryLoadSequenceTests.js",
```

Comment:
Can you add `tests/production/SolutionsRegistryLoadSequenceTests.js` to `tests/production/all-tests.js`?

Reply:
Probably not.  There is a discussion on the architecture list about the security of our GCP clusters and these production tests.  The `SolutionsRegistryLoadSequenceTests` will always fail when run on GCP because GET requests outside of the cluster are not allowed.  If no special exception is made to allow the github request, it is pointless to run the loading sequence tests in GCP.  That's why I took it out of `all-tests.js`, and am adjusting the GCP yaml script to run all the other production tests except the `SolutionsRegistryLoadSequenceTests`.

Reply:
Update:  Stepan has outlined a way to allow outbound requests to github.  I'll see if I can make that work.  If it works, then I'll put it back into `all-tests.js`